bundle/deploy/terraform/init_test.go

```diff
@@ -269,3 +271,119 @@ func TestSetUserProfileFromInheritEnvVars(t *testing.T) {
 	assert.Contains(t, env, "USERPROFILE")
 	assert.Equal(t, env["USERPROFILE"], "c:\\foo\\c")
 }
+
+func TestInheritEnvVarsWithAbsentPluginsCacheDir(t *testing.T) {
+	env := map[string]string{}
+	t.Setenv("DATABRICKS_TF_PLUGIN_CACHE_DIR", "/tmp/cache")
+	err := inheritEnvVars(context.Background(), env)
```

Comment:
Please use `env.Set` with a dedicated context per test. Then we can parallelize.